pkgs/tools/misc/betterdiscordctl/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, pkgs }:
+
+stdenv.mkDerivation rec {
+  pname = "betterdiscordctl";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "bb010g";
+    repo = "betterdiscordctl";
+    rev = "v${version}";
+    sha256 = "0qbas7gbjpi688vmsq4yjbycc0g84zkilsfiikvjmj3nfcmy2xh6";
+  };
+
+  patchPhase = ''
+    substituteInPlace betterdiscordctl --replace "/opt" "${pkgs.discord}/opt"
+  '';
+
+  installPhase = ''
+    install -D betterdiscordctl $out/bin/betterdiscordctl
```

```suggestion
    install -D -m755 betterdiscordctl $out/bin/betterdiscordctl
```
Also, shouldn't the readme and license be installed as well?

---

License not being included is fine. (Probably should strafe it over to ISC rather than MIT anyways.) README is more useful, but it's because I need to get off my ass and actually do the manpage for it, which would cover almost everything the README currently does. And then probably package up [mdoc-to-md](https://github.com/bb010g/mdoc-to-md) for Nixpkgs too, beyond whatever local Nix stuff I've got in the repo.

---

I'll install the readme to `$out/share/doc/betterdiscordctl/`, might as well. Should indeed switch over to manpages when those become available.